src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -84,6 +85,32 @@
         return emails;
     }
     
+    /**
+     * Generates the feedback submission confirmation email for the given {@code session} for {@code student}
+     * and {@code instructor}.
+     */
+    public EmailWrapper generateFeedbackSubmissionConfirmationEmails(
+            FeedbackSessionAttributes session, StudentAttributes student,
+            InstructorAttributes instructor) {
+        CourseAttributes course = coursesLogic.getCourse(session.getCourseId());
+        String template = EmailTemplates.USER_FEEDBACK_SUBMISSION_CONFIRMATION;
+        EmailWrapper email = new EmailWrapper();
+        
+        if (instructor != null) {
+            email = generateFeedbackSessionEmailBaseForInstructorSubmissionConfirmation(course, session,
+                    instructor, template, EmailType.FEEDBACK_SUBMISSION_CONFIRMATION.getSubject());
```

Comment:
You can remove two parameters `template` and `subject` because they are the same in all the call sites of this method.